_layouts/projects.html

```diff
@@ -6,68 +6,76 @@
 		<main id="main" role="main">
 			<section class="project id="">
 				<header class="project--header">
+					<p class="project--tags">
+						<span class="project--type">{{ page.type-of-product | default: "N/A"}}</span> ∙ {{ page.tags }}
+					</p>
 					<h1 class="project--title">{{ page.title }}</h1>
-					<h3 class="project--source-link">
-						<a href="{{ page.link-to-source }}"
+					<div class="project--link">
```

Comment:
We should also wrap this in an `{% if page.link-to-source %}`, just in case we run into one w/o a repo that we still want to post.